.zuul/playbooks/envoy-build/run.yaml

```diff
@@ -0,0 +1,31 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: config-gcc
+      gcc_version: 7
+  tasks:
+    - name: Build envoy
+      shell:
+        cmd: |
+          apt update
+          apt-get install -y \
+             libtool \
+             cmake \
+             automake \
+             autoconf \
+             make \
+             ninja-build \
+             curl \
+             unzip \
+             virtualenv
+
+          # TODO(yikun): Add the logs and test_results preparation in init-test/pre stage,
+          # and add the global env args (LOG_PATH, RESULT_PATH).
+          # See more in: https://github.com/theopenlab/openlab/issues/238
+          mkdir -p '{{ ansible_user_dir }}/workspace/logs'
+          bazel build //source/exe:envoy-static | tee '{{ ansible_user_dir }}/workspace/logs/bazel.txt'
+
+          mkdir -p '{{ ansible_user_dir }}/workspace/test_results'
+          cp -r ./bazel-bin '{{ ansible_user_dir }}/workspace/test_results'
+        chdir: '{{ zuul.project.src_dir }}'
+        executable: /bin/bash
```

Comment:
environment: '{{ global_env }}'

Reply:
I think we don't need this `environment: '{{ global_env }}'` anymore， because we already move var into job.yaml.

Reply:
and we need global_env in containerd job because we need GOPATH. : ), and we don't need global_env in here.

Reply:
Emmm, see the following two formats, if you don't care about ansible, which one you like?
```
bazel build //source/exe:envoy-static | tee '{{ global_env.LOGS_PATH }}/bazel.txt'
```
vs
```
bazel build //source/exe:envoy-static | tee $LOGS_PATH/bazel.txt
```

Reply:
Yes, you are right.
```
bazel build //source/exe:envoy-static | tee $LOGS_PATH/bazel.txt
```
and we need a `environment: '{{ global_env }}'`

Reply:
Thank you.